judges_seed.py

```diff
@@ -0,0 +1,238 @@
+        item.get_text()
+        for index, item in enumerate(get_judge_rows(url))
+        if index % 2 == 0 and item.get_text() != "Judge"
+    ]
+
+
+def get_circuit_district_judges(url: str) -> list[str]:
+    """
+    Retrieves all circuit district judges
+    """
+    rows = get_judge_rows(url)
+    names = []
+    for index, item in enumerate(rows):
+        if index % 3 == 0 and item.get_text() != "Judge":
+            name = item.get_text()
+            if not item.get_text()[-1].isalpha():
+                name = name[:-1]
+            names.append(name)
+    return names
+
+
+def gather_all_judges() -> list[str]:
+    """
+    Retrieves and combines all judges names
+    """
+    high_court_masters = get_diversity_high_court_judges(
+        "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/hc-masters-list/"
+    )
+
+    bench_chairs = get_bench_judges(
+        "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/bench-chairmen-list/"
+    )
+
+    district_judges_magistrates = get_district_judges_magistrates(
+        "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/dj-mags-ct-list/"
+    )
+
+    diversity_community_judges = get_diversity_high_court_judges(
+        "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/diversity-and-community-relations-judges-list/"
+    )
+
+    judge_advocates_general = get_judge_advocates(
+        "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/jag-list/"
+    )
+
+    circuit_judges = get_circuit_district_judges(
+        "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/circuit-judge-list/"
+    )
+
+    district_judges = get_circuit_district_judges(
+        "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/district-judge-list/"
+    )
+
+    diversity_community_magistrates = get_diversity_high_court_judges(
+        "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/diversity-community-and-relations-magistrates/"
+    )
+
+    return (
+        high_court_masters
+        + bench_chairs
+        + district_judges_magistrates
+        + diversity_community_judges
+        + judge_advocates_general
+        + circuit_judges
+        + district_judges
+        + diversity_community_magistrates
+    )
+
+
+def standardise_judge_names(judges: list[str]) -> list[tuple]:
+    """
+    Makes the judges names just the personal name and creates a list of tuples ready for upload
+    """
+    standardised_names = []
+    for judge in judges:
+        name_parts = judge.split(" ")
+        good_parts = []
+        for part in name_parts:
+            if part not in COMMON_JUDGE_WORDS and part[0].isalpha():
```

Comment:
I love this solution to our problem of titles, beautiful and simple, willingness to get dirty and look at raw data we are processing and how it's coming out, great observation.